.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+#### Summary
+[todo]
+<!--
+  A description of this pull request.
+-->
+
+#### Jira ticket
+- MM-Txxxxx
+
+#### Screenshots of the modals or screens in all target clients (required)
+- [] In Web
+- [] In Desktop app
+- [] In Mobile app, both Android and iOS
```

Comment:
Are there guidelines for how or where I can capture screenshots for new notices?

Reply:
Correct, it's not implemented on mobile yet